docs/apache-airflow/core-concepts/operators.rst

```diff
@@ -211,64 +235,65 @@ Alternatively, if you want to prevent Airflow from treating a value as a referen
 Rendering Fields as Native Python Objects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, all the ``template_fields`` are rendered as strings.
-
-Example, let's say ``extract`` task pushes a dictionary
-(Example: ``{"1001": 301.27, "1002": 433.21, "1003": 502.22}``) to :ref:`XCom <concepts:xcom>` table.
-Now, when the following task is run, ``order_data`` argument is passed a string, example:
-``'{"1001": 301.27, "1002": 433.21, "1003": 502.22}'``.
+By default, all Jinja templates in ``template_fields`` are rendered as strings. This however not always desired. For example, let's say an ``extract`` task pushes a dictionary ``{"1001": 301.27, "1002": 433.21, "1003": 502.22}`` to :ref:`XCom <concepts:xcom>`:
```

Comment:
This however **is** not always desired

Reply:
Hey , I do not understand what you mean , could you explicitly say what needs to be replace by what , thanks

Reply:
The sentence "_This however not always desired_." is grammatically incorrect, it needs an "is".
(It is probably better to add commas around _however_ in that sentence, but that's not as important IMHO.)